### ISSUE TITLE
Matplotlib can't read its configuration file on non-ascii Windows  accounts

### ISSUE THREAD

**Issue by ccordoba12** [matplotlib/matplotlib#3757]
I'm the Spyder current maintainer. In our last release we added a couple of features related to Pandas. Unfortunately, several users are reporting now that Spyder crashes under non-ascii Windows accounts. That's because importing Pandas imports Matplotlib,  and Matplotlib fails while trying to read its config file.

This is the traceback reported by one of our users, where this can be seen clearly

```
C:\Users\Damián>spyder
Traceback (most recent call last):
  File "C:\Anaconda\Scripts\spyder-script.py", line 1, in <module>
    from spyderlib import start_app
  File "C:\Anaconda\lib\site-packages\spyderlib\start_app.py", line 14, in <module>
    from spyderlib.config import CONF
  File "C:\Anaconda\lib\site-packages\spyderlib\config.py", line 22, in <module>

    from spyderlib.utils import iofuncs, codeanalysis
  File "C:\Anaconda\lib\site-packages\spyderlib\utils\iofuncs.py", line 26, in <module>
    import pandas as pd
  File "C:\Anaconda\lib\site-packages\pandas\__init__.py", line 41, in <module>
    from pandas.core.api import *
  File "C:\Anaconda\lib\site-packages\pandas\core\api.py", line 9, in <module>
    from pandas.core.groupby import Grouper
  File "C:\Anaconda\lib\site-packages\pandas\core\groupby.py", line 15, in <module>
    from pandas.core.frame import DataFrame
  File "C:\Anaconda\lib\site-packages\pandas\core\frame.py", line 38, in <module>
    from pandas.core.series import Series
  File "C:\Anaconda\lib\site-packages\pandas\core\series.py", line 2524, in <module>
    import pandas.tools.plotting as _gfx
  File "C:\Anaconda\lib\site-packages\pandas\tools\plotting.py", line 26, in <module>
    import pandas.tseries.converter as conv
  File "C:\Anaconda\lib\site-packages\pandas\tseries\converter.py", line 7, in <module>
    import matplotlib.units as units
  File "C:\Anaconda\lib\site-packages\matplotlib\__init__.py", line 1048, in <module>
    rcParams = rc_params()
  File "C:\Anaconda\lib\site-packages\matplotlib\__init__.py", line 897, in rc_params
    fname = matplotlib_fname()
  File "C:\Anaconda\lib\site-packages\matplotlib\__init__.py", line 748, in matplotlib_fname
    fname = os.path.join(os.getcwd(), 'matplotlibrc')
  File "C:\Anaconda\lib\ntpath.py", line 84, in join
    result_path = result_path + p_path
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe1 in position 11: ordinal not in range(128)
```

I know, I know, why on earth Windows decided to let users define non-ascii accounts?!! We had to deal with this a couple of years ago, so if you need help with it please let me know :-)

**Comment by miguelbalmeida** [comment]
This problem exists in the current version of Anaconda (2.1.0) which uses matplotlib 1.4.0.
I can confirm that Anaconda 2.0.1 uses matplotlib 1.3.1 and does not have this problem.

(Just wanted to add this for people who had this problem with Anaconda, like myself)

**Comment by jenshnielsen** [comment]
@miguelbalmeida you should be able to use the ```conda``` command in anaconda to upgrade matplotlib to `1.4.2` where this issue is resolved. In general a number of issues have been resolved between 1.4.0 and 1.4.2

**Comment by miguelbalmeida** [comment]
@jenshnielsen thanks for the tip. We ended up going with Anaconda 2.0.1 anyway to make sure it was easy to install.
